setup.py

```diff
@@ -8,8 +8,8 @@
 here = path.abspath(path.dirname(__file__))
 
 requires = [
-    'botocore>=1.3.15',
-    'awscli>=1.14.10'
+    'botocore>=1.15.20',
+    'awscli>=1.18.20'
```

Comment:
Unless I'm mistaken, nothing you've added here requires a newer version of either the awscli or boto. If you have a more recent version, this should still work with them fine.

Reply:
you are right. reverted this change.